package.json

```diff
@@ -38,7 +38,7 @@
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
     "nano-staged": "^0.5.0",
-    "size-limit": "^8.0.1",
+    "size-limit": "^7.0.8",
```

Comment:
Without this change, `npm i` fails with:

```console
% npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: tinybench@2.3.1
npm ERR! Found: size-limit@8.2.4
npm ERR! node_modules/size-limit
npm ERR!   dev size-limit@"^8.2.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer size-limit@"7.0.8" from @size-limit/preset-small-lib@7.0.8
npm ERR! node_modules/@size-limit/preset-small-lib
npm ERR!   dev @size-limit/preset-small-lib@"^7.0.4" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/alex/.npm/_logs/2023-02-21T11_08_37_616Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alex/.npm/_logs/2023-02-21T11_08_37_616Z-debug-0.log
```

If you update `size-limit`, `@size-limit/preset-small-li` and `@size-limit/time` all to the latest versions, conflicting versions of `esbuild` are pulled in which `tsup` tries to pass the `watch` flag to - this was recently removed and errors out, so the simplest thing to do here seems to be to downgrade `size-limit`.